### PR TITLE
Add 1.0 changelog for v1.0.2

### PR DESCRIPTION
**Pull request by msau42**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
We didn't have a changelog file for 1.0.x, so adding a new file and also included older v1.0.1 changelog history as well using the old format.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG-1.0.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Changes since v1.0.1
+
+## New Features
+
+- Users can now provide a secret name and namespace during provision by passing the correct storage class parameters: "provisioner-secret-name" and "provisioner-secret-namespace" ([#281](https://github.com/kubernetes-csi/external-provisioner/pull/281), [@oleksiys](https://github.com/oleksiys))
```
Comment (davidz627) [inline CHANGELOG-1.0.md]: new feature in a patch release?

Comment (msau42) [reply CHANGELOG-1.0.md]: yeah this one is low risk

Comment (saad-ali) [inline CHANGELOG-1.0.md]: We should be specific here `csi.storage.k8s.io/provisioner-secret-name` and `csi.storage.k8s.io/provisioner-secret-namespace`

Comment (msau42) [reply CHANGELOG-1.0.md]: fixed

**Pull request merged**